scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -3,56 +3,101 @@ package org.oppia.android.scripts.coverage
 import org.oppia.android.scripts.proto.Coverage
+import org.oppia.android.scripts.proto.CoverageDetails
+import org.oppia.android.scripts.proto.CoverageExemption
+import org.oppia.android.scripts.proto.CoverageFailure
 import org.oppia.android.scripts.proto.CoverageReport
+import org.oppia.android.scripts.proto.CoverageReportContainer
 import org.oppia.android.scripts.proto.CoveredLine
 import org.oppia.android.scripts.proto.TestFileExemptions
 import java.io.File
 import java.util.concurrent.TimeUnit
 
+/** ANSI escape codes for colors. */
+/** Green text. */
+const val GREEN = "\u001B[32m"
+/** Red text. */
+const val RED = "\u001B[31m"
+/** Default text. */
+const val RESET = "\u001B[0m"
+/** Bold text. */
+const val BOLD = "\u001B[1m"
+
 /**
  * Entry point function for running coverage analysis for a source file.
  *
  * Usage:
- *   bazel run //scripts:run_coverage_for_test_target -- <path_to_root> <relative_path_to_file>
+ *    bazel run //scripts:run_coverage_for_test_target -- <path_to_root> <list_of_relative_path_to_files>
  *
  * Arguments:
  * - path_to_root: directory path to the root of the Oppia Android repository.
- * - relative_path_to_file: the relative path to the file to analyse coverage
- * - reportFormat: the format of the coverage report. Defaults to MARKDOWN if not specified.
- *   Available options: MARKDOWN, HTML.
+ * - list_of_relative_path_to_files: the list of relative path to the files to analyse coverage
+ * - reportFormat: the format of the coverage report. Defaults to HTML if not specified.
+ *    Available options: MARKDOWN, HTML.
+ * - processTimeout: The amount of time that should be waited before considering a process as 'hung',
+ *    in minutes.
+ * - path_to_output_file: path to the file in which the collected coverage reports will be printed.
  *
  * Example:
- *     bazel run //scripts:run_coverage -- $(pwd)
- *     utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt format=HTML
+ *    bazel run //scripts:run_coverage -- $(pwd)
+ *    utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt --format=HTML
+ *
+ * Example with list of files:
+ *    bazel run //scripts:run_coverage -- $(pwd)
+ *    utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt
+ *    utility/src/main/java/org/oppia/android/util/math/MathTokenizer.kt --format=MARKDOWN
+ *
  * Example with custom process timeout:
- *     bazel run //scripts:run_coverage -- $(pwd)
- *     utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt processTimeout=15
+ *    bazel run //scripts:run_coverage -- $(pwd)
+ *    utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt --processTimeout=15
  *
+ * Example with output path to save the collected coverage proto:
+ *    bazel run //scripts:run_coverage -- $(pwd)
+ *    utility/src/main/java/org/oppia/android/util/parser/math/MathModel.kt
+ *    --protoOutputPath=/tmp/coverage_report.proto64
```

Comment:
Perhaps something that might be cleaner is just to introduce a proto64 format (as a choice other than HTML/MD)? That would also simplify the script a bit since you shouldn't need to bother with MD/HTML output if the only thing that's cared about is the proto.

Reply:
Ah, yeah! Thanks, Ben. I've introduced the **PROTO** report format, with that, it skips the report generation executions.

Reply:
Added / modified 2 test cases to test this behaviour,
- testRunCoverage_withProtoReportFormat_savesCoverageContainerProto
- testRunCoverage_withProtoReportFormat_skipsCoverageGeneration